### DISCUSSION TITLE
Resticprofile 0.18 & Windows Task Schedule

### DISCUSSION THREAD

**Discussion by ryszard-suchocki** [creativeprojects/resticprofile#141] [Q&A] [answered]
Hello,
I want to ask more experienced users, how to schedule a backup in Windows. 

My configuration is:
schedule: 'mon...fri 7:30,9,11,13,15,17,19' 

My intention is to create a backup at 7:30 and recur it at 2h intervals. Currently 'schedule' actions give me backups at 7:11, 7:13, 7:15, 7:17. What is wrong with my configuration? 

Analyzing backup schedule 1/1
=================================
  Original form: mon..sat 07:30,11,13,15,17,19
Normalized form: Mon..Sat *-*-* 07:11,13,15,17,19,30:00
    Next elapse: Tue Sep  6 07:11:00 CEST 2022
       (in UTC): Tue Sep  6 05:11:00 UTC 2022
       From now: 19h53m42s left

**Comment by creativeprojects** [comment, marked as answer]
You need to specify the hours together, then the minutes:

So what you need is this one instead: `mon..fri 7,9,11,13,15,17,19:30`

**Reply by ryszard-suchocki** [reply]
Thanks!!!
